autoscaler/controllers/datacollection/configmap.go

```diff
@@ -139,6 +145,24 @@ func getConfigMapData(apps *odigosv1.InstrumentedApplicationList, dests *odigosv
 	}
 	processorsCfg["resourcedetection"] = config.GenericMap{"detectors": []string{"ec2", "gcp", "azure"}}
 
+	exporters := config.GenericMap{
+		"otlp/gateway": config.GenericMap{
+			"endpoint": fmt.Sprintf("dns:///odigos-gateway.%s:4317", env.GetCurrentNamespace()),
+			"tls": config.GenericMap{
+				"insecure": true,
+			},
+		},
+	}
+	tracesPipelineExporter := []string{"otlp/gateway"}
+
+	if setTracesLoadBalancer {
+		exporters["loadbalancing"] = config.GenericMap{
+			"protocol": config.GenericMap{"otlp": config.GenericMap{"tls": config.GenericMap{"insecure": true}}},
+			"resolver": config.GenericMap{"k8s": config.GenericMap{"service": fmt.Sprintf("odigos-gateway.%s", env.GetCurrentNamespace())}},
```

Comment:
I wonder if we should maybe use dns resolver instead of k8s. 
1. There may be a lot of data collection collectors running (daemonset..) and it may cause additional load on Kubernetes API server
2. Requires additional permissions (endpoints)

I guess k8s updates faster than dns but I wonder if the speed worth the downsides (I don't have strong opinion, just raising questions 😀)
cc @blumamir